BuildResidentialHPXML/measure.rb

```diff
@@ -154,26 +154,34 @@ def arguments(model)
     level_choices << 'Middle'
     level_choices << 'Top'
 
-    arg = OpenStudio::Measure::OSArgument::makeChoiceArgument('geometry_unit_level', level_choices, false)
-    arg.setDisplayName('Geometry: Unit Level')
-    arg.setDescription("The level of the unit. This is required for #{HPXML::ResidentialTypeApartment}s.")
+    arg = OpenStudio::Measure::OSArgument::makeBoolArgument('geometry_unit_left_wall_is_adiabatic', true)
+    arg.setDisplayName('Geometry: Unit Left Wall Is Adiabatic')
+    arg.setDescription('Presence of an adiabatic left wall.')
+    arg.setDefaultValue(false)
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeBoolArgument('geometry_unit_right_wall_is_adiabatic', true)
+    arg.setDisplayName('Geometry: Unit Right Wall Is Adiabatic')
+    arg.setDescription('Presence of an adiabatic right wall.')
+    arg.setDefaultValue(false)
     args << arg
 
-    horizontal_location_choices = OpenStudio::StringVector.new
-    horizontal_location_choices << 'None'
-    horizontal_location_choices << 'Left'
-    horizontal_location_choices << 'Middle'
-    horizontal_location_choices << 'Right'
+    arg = OpenStudio::Measure::OSArgument::makeBoolArgument('geometry_unit_front_wall_is_adiabatic', true)
+    arg.setDisplayName('Geometry: Unit Front Wall Is Adiabatic')
+    arg.setDescription('Presence of an adiabatic front wall.')
```

Comment:
I suggest we add some description that a front wall might be adiabatic if, e.g., it is adjacent to a conditioned corridor.